octobot/configuration_manager.py

```diff
@@ -143,14 +144,20 @@ def set_default_profile(config, from_default_config_file=constants.DEFAULT_CONFI
     config.save()
 
 
-def get_default_tentacles_url(version=None):
-    if version is None:
+def get_default_tentacles_url(version=None, use_beta_tentacles=False):
```

Comment:
Couldn't we use `version="beta"` instead of `use_beta_tentacles`?

Reply:
Right, it will be simpler, I added it because it uses another repository but in the end the user of this method doesn't have to know it, better keeping it simple